tests/integration/mixer/outboundtrafficpolicy/v2/traffic_registry_only_test.go

```diff
@@ -58,7 +56,7 @@ func TestOutboundTrafficPolicy_RegistryOnly_TelemetryV2(t *testing.T) {
 			Gateway:  true,
 			Expected: outboundtrafficpolicy.Expected{
 				Metric:          "istio_requests_total",
-				PromQueryFormat: `sum(istio_requests_total{destination_service_name="istio-egressgateway",response_code="200"})`,
+				PromQueryFormat: `sum(istio_requests_total{destination_service_name="istio-egressgateway.istio-system.svc.cluster.local",response_code="200"})`,
```

Comment:
@nrjpoddar @mandarjog This was a change I noticed from mixer to telemetry v2 labels.

For mixer, the label created was `istio-egressgateway` and it's the fqdn in telemetry v2.